tests/pylint/censorship.py

```diff
@@ -116,6 +117,23 @@ def run(self):
     def _prepare_args(self):
         args = []
 
+        # pylint uses a lot of memory and doesn't handle ENOMEM well, so set --jobs based on memory
```

Comment:
Do you think it might be worth it to report the issue to pylint?

Reply:
https://github.com/PyCQA/pylint/issues/1495 has existed for years, but I'll report a more specific one.